application/controllers/admin/Tokens.php

```diff
@@ -1688,7 +1688,8 @@ public function email(int $iSurveyId)
 
                     unset($fieldsarray);
                 }
-
+                // Closes a still active SMTP connection if it exists
+                $mail->smtpClose();
```

Comment:
Didn't broke if mailer are not smtp ?

Reply:
Shouldn't because the function checks if smtp object exists.

Reply:
Not sure if this is even needed as smtpClose is also done in the PHPMailer desctructor by default.